ph5/utilities/ph5validate.py

```diff
@@ -436,6 +437,11 @@ def check_station_completeness(self, station):
         e, w = validation.check_lat_lon_elev(station)
         error = error + e
         warning = warning + w
+        if station['location/Z/units_s'] in ['unknown',
+                                             'UNKNOWN']:
+            warning.append("location/Z/units_s is set " +
+                           "unknown. Consider updating " +
+                           "this unit to m.")
```

Comment:
I think these lines should be moved in under validation.py.check_lat_lon_elev().
To include all combinations of 'unknown', the condition should be changed to:
`if station['location/Z/units_s'].lower() == 'unknown':`